README.md

```diff
@@ -1,4 +0,0 @@
-# Introduction
```

Comment:
@AustinBwanika Restore the original file as it was since I see you changed it here